bin/web

```diff
@@ -7,6 +7,10 @@ ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/.. && pwd )"
 DEV_PORT=8080
 export NODE_ENV=${NODE_ENV:=development}
 
+# Stop everything in the process group (in the background) whenever the
+# parent process experiences an error and exits.
+trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
```

Comment:
**Nice**. This forced me to dig into quite some bash docs to fully understand, and it's awesome.
I also realized that even when there are no errors, when you killed `bin/web dev` the processes for `kubectl port-forward` remained in the background, and this fixes that as well :+1:

Reply:
Turns out CI is not happy about it because when `/bin/web` finishes it returns status 0.
Getting rid of the `EXIT` in this trap fixes that, but it seems that interferes with trapping the errors from other scripts (maybe because of `set -e` in those other scripts?).
Not sure what to do in this case. Maybe it's not necessary for CI to run a naked `/bin/web`...

Reply:
@alpeb I've been playing around and seeing what to do, the problem is that it signals the parent as well as the children. Just need to figure out how to limit to the children only...